cloud-azure/src/main/java/com/sequenceiq/cloudbreak/cloud/azure/AzureCredentialConnector.java

```diff
@@ -72,13 +54,10 @@ public CloudCredentialStatus verify(AuthenticatedContext authenticatedContext, C
                     return new CloudCredentialStatus(authenticatedContext.getCloudCredential(), CredentialStatus.FAILED, null,
                             "Your subscription ID is not valid");
                 } else {
-                    client.getRefreshToken()
-                            .ifPresent(refreshToken -> {
-                                Map<String, String> codeGrantFlowBased = (Map<String, String>) cloudCredential
-                                        .getParameter("azure", Map.class)
-                                        .get(AzureCredentialView.CODE_GRANT_FLOW_BASED);
-                                codeGrantFlowBased.put("refreshToken", refreshToken);
-                            });
+                    Optional<String> accessToken = client.getAccessToken();
+                    if (accessToken.isPresent()) {
+                        LOGGER.info("Azure returned access token.");
```

Comment:
Do we need this log line, I would rather log the error path probably would end up in less logs.

Reply:
Actually it will throw an exception.